src/components/frame/TokenActivation/TokensTab/TokensTab.tsx

```diff
@@ -0,0 +1,234 @@
+import TokenTooltip from './TokenTooltip';
+import SmallTokenAmountMessage from './SmallTokenAmountMessage';
+
+const displayName = 'frame.TokenActivation.TokensTab';
+
+const MSG = defineMessages({
+  active: {
+    id: `${displayName}.active`,
+    defaultMessage: 'Active',
+  },
+  activeLocked: {
+    id: `${displayName}.activeLocked`,
+    defaultMessage: 'Active',
+  },
+  staked: {
+    id: `${displayName}.staked`,
+    defaultMessage: 'Staked',
+  },
+  inactive: {
+    id: `${displayName}.inactive`,
+    defaultMessage: 'Inactive',
+  },
+  activeTokensTooltip: {
+    id: `${displayName}.activeTokensTooltip`,
+    defaultMessage: `Tokens are “Active” when they’ve been deposited to a
+      contract which lets them get ‘locked’ when you need to stake,
+      or claim a share of Rewards. You can withdraw tokens back
+      to your wallet any time, you just need to clear any locks first.`,
+  },
+  inactiveTokensTooltip: {
+    id: `${displayName}.inactiveTokensTooltip`,
+    defaultMessage: `Inactive tokens are contained in your own wallet.
+      You need to “Activate” them to stake, or be eligible to receive Rewards.`,
+  },
+  stakedTokensTooltip: {
+    id: `${displayName}.stakedTokensTooltip`,
+    defaultMessage: `You have tokens staked in processes which must conclude
+      before they can be deactivated.`,
+  },
+  pendingError: {
+    id: `${displayName}.pendingError`,
+    defaultMessage: 'Error: balance pending!',
+  },
+  pendingErrorTooltip: {
+    id: `${displayName}.pendingErrorTooltip`,
+    defaultMessage: 'Send any “Activate” transaction to claim pending balance.',
+  },
+});
+
+export interface TokensTabProps {
+  activeTokens: BigNumber;
+  inactiveTokens: BigNumber;
+  totalTokens: BigNumber;
+  lockedTokens: BigNumber;
+  isPendingBalanceZero: boolean;
+  token: Token;
+}
+
+const TokensTab = ({
+  activeTokens,
+  inactiveTokens,
+  totalTokens,
+  lockedTokens,
+  token,
+  isPendingBalanceZero,
+}: TokensTabProps) => {
+  const { colony } = useColonyContext();
+  const targetRef = useRef<HTMLParagraphElement>(null);
+
+  const [totalTokensWidth, setTotalTokensWidth] = useState(0);
+
+  const widthLimit = 164;
+
+  useLayoutEffect(() => {
+    if (targetRef?.current && totalTokensWidth === 0) {
+      setTotalTokensWidth(targetRef?.current?.offsetWidth);
+    }
+  }, [totalTokensWidth]);
+
+  const hasLockedTokens = useMemo(() => !lockedTokens.isZero(), [lockedTokens]);
+
+  const tokenDecimals = useMemo(
+    () => getTokenDecimalsWithFallback(token?.decimals),
+    [token],
+  );
+
+  const formattedTotalAmount = getFormattedTokenValue(
+    totalTokens,
+    token.decimals,
+  );
+  const formattedLockedTokens = getFormattedTokenValue(
+    lockedTokens,
+    token.decimals,
+  );
+  const formattedActiveTokens = getFormattedTokenValue(
+    activeTokens,
+    token.decimals,
+  );
+  const formattedInactiveTokens = getFormattedTokenValue(
+    inactiveTokens,
+    token.decimals,
+  );
```

Comment:
All this logic doesn't really make sense in the context of us having data readily available from within a database. 

I guess it's good enough for now, since all data is mocked, but once the real data comes in, this needs to be removed/refactored